### PR TITLE
Separated flap and slats sound, made new taxi rattles, and more

### PR DESCRIPTION
**Pull request by hotshotp**
## Summary of Changes

I added a lot of changes to the sounds, so here are the minor ones:

- Made wind sound louder on approach (have a steeper volume curve)
- Made engine rumble at high thrust values louder (and more powerful)
- Slightly altered the battery and external power relay sounds to be less "plasticky"

Now for the major changes:

- Made exterior fuel pump sounds when close to the wing
- Separated flap and slat sounds (no sound for slats)
- Completely reworked touchdown sounds: 
When you land at under 50 fpm no touchdown sound plays
When you land between 50 and 100 fpm there will be a soft rattle ish sound
When you land between 101 and 350 fpm there will be an audible click-thump sound
When you land at over 351 you will hear a thud, squeak, and loud rattle sound
- Added a completely new taxi rattle sound which of course gets faster with speed
- And finally made spoilers drag sounds

## References

I have taken sounds from IRL samples and made my own (such as the wind) and took feedback from several IRL pilots and engineers. New taxi sound from PaRaBas.

## Additional context

Discord username (if different from GitHub):

Boris#9134

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml [hunk 1]
```diff
@@ -62,15 +59,26 @@
 
     <MiscellaneousSounds>
         <Sound WwiseEvent="AP_PREFLIGHT_CHECK_OVER" ViewPoint="Inside" WwiseData="true" />
-        <Sound WwiseData="false" WwiseEvent="FLAPS" FileName="FINALFLAPS" NodeName="WING_AILERON_1_LEFT" FullScaleDistance="4" Continuous="true" FadeOutType="2" FadeOutTime="0.5" />
-        <Sound WwiseData="false" WwiseEvent="FLAPS" FileName="FINALFLAPS" NodeName="WING_AILERON_1_RIGHT" FullScaleDistance="4" Continuous="true" FadeOutType="2" FadeOutTime="0.5" />
+        
+        
+
```
Comment (Benjozork) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:65]:
```suggestion
```

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml [hunk 2]
```diff
@@ -59,8 +59,8 @@
 
     <MiscellaneousSounds>
         <Sound WwiseEvent="AP_PREFLIGHT_CHECK_OVER" ViewPoint="Inside" WwiseData="true" />
-        
-        
+
+
```
Comment (Benjozork) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml]: Why so many empty lines ?

**Comment by Awemeter** [comment]
ground roll sounds start going fast around 30 kts
good thing
no slat sounds 
good thing
spoiler drag sounds sound good
good thing
more rumble
good thing

**Comment by NathanInnes** [comment]
Do we have a credits system in place for all the sound work?

**Comment by hotshotp** [comment]
> Do we have a credits system in place for all the sound work?

i didn't take any sounds off of youtube for this pr, when i do i will make sure to credit the youtube videos later, i already had the talk

**Comment by luke-stec** [comment]
All sounds good to me!

**Comment by z0rgmeister** [comment]
Name: z0rgmeister#8972
Date of testing: 04/11/2020
Version of the sim: 1.10.8.0
PR Tested: #1751
Object of test: To test sounds as per PR
Steps taken: 

1.Circuit with takeoff and 4x stop and go landings at RWY27 EGGD.
2.Every landing as per PR to test different touchdown sounds (<50 fpm; 50-100 fpm; 101-351 and >351 fpm). All verified working as PR.
3.Full stop on the runway.

Flight notes: Realtime weather winds. Started from C&D. Verified external fuel pump sounds as per PR.

Taxi/Takeoff : Verified taxi sounds at different speeds as per PR. Engine high N1 low end rumble audible as per PR.

Climb/Cruise: N/A. Leveled out at 3000 MSL.

Descent/Approach/Landing: Different touchdown speeds on different touch and gos to verify new touchdown sounds. Audible as per PR. Slight issue, no internal or external flap sounds noted.

Shutdown: NA

Issues: Flap sounds (1-full) not audible.

Overall Rating: Good, some notes.

Conclusions of the testing: Good job on the new sounds, especially that fuel pump sound and the new oomph from the engine low end. However not hearing the flaps makes me wonder if it was some glitch in the matrix. Maybe there's something missing somewhere? Anyway, should be easily fixable.

**Comment by hotshotp** [comment]
> Name: z0rgmeister#8972
> Date of testing: 04/11/2020
> Version of the sim: 1.10.8.0
> PR Tested: #1751
> Object of test: To test sounds as per PR
> Steps taken:
> 
> 1.Circuit with takeoff and 4x stop and go landings at RWY27 EGGD.
> 2.Every landing as per PR to test different touchdown sounds (<50 fpm; 50-100 fpm; 101-351 and >351 fpm). All verified working as PR.
> 3.Full stop on the runway.
> 
> Flight notes: Realtime weather winds. Started from C&D. Verified external fuel pump sounds as per PR.
> 
> Taxi/Takeoff : Verified taxi sounds at different speeds as per PR. Engine high N1 low end rumble audible as per PR.
> 
> Climb/Cruise: N/A. Leveled out at 3000 MSL.
> 
> Descent/Approach/Landing: Verified increase in wind sound as per PR. Different touchdown speeds on different touch and gos to verify new touchdown sounds. Audible as per PR.
> 
> Shutdown: NA
> 
> Issues: Flap sounds (1-full) not audible.
> 
> Overall Rating: Good,
> 
> Conclusions of the testing: Good job on the new sounds, especially that fuel pump sound and the new oomph from the engine low end. EDIT: Flap sounds are there in wing view.

flaps are only audible in the wing view just like intended

**Comment by danbw9** [comment]
Name: danbw9#7028
Date of testing: 04/11/2020
Version of the sim: 1.10.8.0
PR Tested: #1751
Object of test: Test various sounds 
Steps taken:

One circuit at UAAA to check the obvious changes and any strange anomalies. 

Completed normal flight from EDDF to UAAA. 

In drone view, confirmed the more audible sound of the fuel pumps whilst by the wing and also in EXT View 11 and 12 fixed cameras from in cabin. 

Will continue to test in next couple days if PR not merged yet...just to get in more landings and compare the touch down sounds. Will report back if find anything weird. 

Flight notes: 

Taxi/Takeoff : Long taxi at EDDF and taxiing at different speeds to check the audible rattles. Engine high N1 low end rumble all correct. 

Climb/Cruise: Coordinated with Boris over voice - listening to speed brake drag sounds at high and low speeds (280 and 210 knots)

Descent/Approach/Landing: Sound is balanced correctly with the wind sounds on approach along with gear down drag sounds. 

Shutdown: NA

Issues: None.
Overall Rating: Good.
Conclusions of the testing: SGTM. Great job Boris. No issues from my side.

**Comment by Benjozork** [comment]
LGTM

**Review by Benjozork — APPROVED** [review]

**Comment by Watsi01** [comment]
@hotshotp "When you land at over 351 you will hear a thud, squeak, and loud rattle sound"

- When you land over 700 you will hear bones brake and a "Yeehaaa!!!" from the cabine.

**Pull request merged**